apns: update T Star, Vibo and AURORA

a0193143 opened the pull request.

SirRGB commented:
can you please drop the unneeded commit and get proper authorship done?
but thx for your contribution.

you also seem to have shortened the apn list, is there any reason for dropping vibo and aurora?

a0193143 committed:
apns: update T Star, Vibo and AURORA

Fix T Star not setup appropriately ( Vibo and AURORA was merged by T Star ), and update protocols for IPv6.

a0193143 commented:
@SirRGB Sorry for the incovinient, I was not familiar with git, now fixed.
The reason of dropping Vibo and Aurora is because they were merged to T Star years ago.

SirRGB commented:
Thx a lot for your contribution
